src/sbt-test/settings/lazy-protoc-resolution/build.sbt

```diff
@@ -1 +1,20 @@
-PB.protocDependency := "non" % "existing" % "artifact"
+// PB.targets, no proto
+lazy val a = project
+  .settings(
+    PB.protocVersion := "1.2.3+non-existing-a",
```

Comment:
Considerations for making the test fail if protoc is fetched:
- I used an invalid `protocVersion` rather than an invalid `protocDependency` as I thought `protocDependency` might become become looked up at a higher level than the project one day (for caching?)
- I chose to set a specific value scoped at the project level rather than for `ThisBuild` (even though `protocVersion` default is currently global), to avoid potential false negatives if that default moves to the project level at some point